### ISSUE TITLE
Where is the categorical vector?

### ISSUE THREAD

**Issue by Steve-Tod** [WangYueFt/dgcnn#45]
Hi, thank you for sharing the code. In your paper, there is a categorical vector for semantic segmentation network. 
<img width="835" alt="image" src="https://user-images.githubusercontent.com/26958167/71346851-d87a1d80-25a3-11ea-9d54-33b3ec29bad7.png">

However, I didn't find many descriptions of this vector in the paper or any implementation of this in the code.
https://github.com/WangYueFt/dgcnn/blob/69b80a370df4125adb6ac376169de2de76a61d67/tensorflow/sem_seg/model.py#L87
Here, after the max pooling, the global feature is directly repeated and concatenated with point features.

What is this categorical vector? Is it necessary?

**Comment by Steve-Tod** [comment]
Sorry, I found this vector is for part segmentation. 

Another question, in PyTorch version conv2d without bias is used but in TensorFlow version conv2d with bias is used. Why there is such a difference?

**Comment by WangYueFt** [comment]
> Sorry, I found this vector is for part segmentation.
> 
> Another question, in PyTorch version conv2d without bias is used but in TensorFlow version conv2d with bias is used. Why there is such a difference?

With batchnormalization, w/ or w/o bias doesn't make any difference.
